src/components/Application/SectionWrapper.tsx

```diff
@@ -31,30 +36,33 @@ const SectionWrapper: React.FC<SectionProps> = ({
     <Accordion styled fluid>
       <Segment.Group size="large">
         <Accordion.Title active={isOpen} onClick={handleClick}>
-          <Grid>
-            <Grid.Row>
-              <Grid.Column width={15}>
-                <Header as="h2" content={details.title} />
-              </Grid.Column>
-              <Grid.Column width={1}>
-                <Icon name={isOpen ? 'angle up' : 'angle down'} size="large" />
-              </Grid.Column>
-            </Grid.Row>
+          <Grid columns="equal">
+            <Grid.Column floated="left">
+              <Header as="h2" content={details.title} />
+            </Grid.Column>
+            <Grid.Column floated="right" textAlign="right">
+              {extraSectionTitleContent && extraSectionTitleContent(section)}
+            </Grid.Column>
+            <Grid.Column floated="right" textAlign="right" width={1}>
```

Comment:
```suggestion
            <Grid.Column>
              <Header as="h2" content={details.title} />
            </Grid.Column>
            <Grid.Column floated="right" textAlign="right">
              {extraSectionTitleContent && extraSectionTitleContent(section)}
            </Grid.Column>
            <Grid.Column floated="right" textAlign="right">
```

Similar layout problem (on mobile) found here.

Reply:
i feel like the progress bar is shifted too much to the left this way. hmm, was really hoping to sort out styling towards the end (i mean if you check out the rest of the app in iPhone 5 mode...), the reason I even changed them in this pr was to simplify jsx. How do you feel about leaving this one for fixing layout stage stage ?